atlasdb-workload-server/src/main/java/com/palantir/atlasdb/workload/workflow/DefaultWorkflow.java

```diff
@@ -60,7 +61,7 @@ public static <T extends TransactionStore> Workflow create(
     @Override
     public WorkflowHistory run() {
```

Comment:
We may wish to update the `WorkflowHistory` class that it only accepts `FullyWitnessedTransaction` rather than both classes. I think this is ok for now.

Reply:
Yep. Could we leave a todo here? `WorkflowHistory` should only accept `FullyWitnessedTransaction` long term, yeah.